src/drill/drill.js

```diff
@@ -0,0 +1,17 @@
+const React = require('react');
+const omit = require('lodash.omit');
+const { shallow } = require('enzyme');
+
+const shallowRenderProp = (wrapper, expander) =>
+  typeof expander === 'string'
+    ? wrapper.prop(expander)()
+    : expander(wrapper.props());
+
+module.exports = function renderProp(expander) {
+  // throws an error if the current instance has a length other than one.
+  return this.single('renderProp', () =>
```

Comment:
...this too?

Reply:
Yep, fixed them both in 8922e7f2d28dbc06d449094d444efad852365464